docs/gen_user_guide.py

```diff
@@ -2,17 +2,20 @@
 import subprocess
 from pathlib import Path
 
-docs_dir = Path(__file__).parent
-subprocess.run(
-    [
-        "pandoc",
-        "-f",
-        "markdown",
-        "-t",
-        "html5",
-        "-o",
-        f"{docs_dir}/user_guide.html",
-        f"{docs_dir}/user_guide.md",
-    ],
-    check=True,
-)
+
+def generate_html() -> None:
+    """Converts user_guide.md to html format."""
+    docs_dir = Path(__file__).parent
+    subprocess.run(
+        [
+            "pandoc",
+            "-f",
+            "markdown",
+            "-t",
+            "html5",
+            "-o",
+            f"{docs_dir}/user_guide.html",
+            f"{docs_dir}/user_guide.md",
+        ],
+        check=True,
+    )
```

Comment:
I'd keep this script so it can also be run manually outside of `FINESSE.spec`. Just add:

```py
if __name__ == "__main__":
    generate_html()
```